### DISCUSSION TITLE
httprunner  V1.5.8的一个问题，求大佬解答

### DISCUSSION THREAD

**Discussion by christ199705** [httprunner/httprunner#1383] [2 - 答疑（Q&A）]
本地搭建运行没问题，但是部署到dokcer 容器中，httprunner对象的run方法执行用例时超时

![image](https://user-images.githubusercontent.com/94459616/175484140-16fa46ac-b50c-48a4-9d92-f2cc38d7713f.png)

**Comment by debugtalk** [comment]
@christ199705 

看上去是网络的问题，你验证下 docker 容器中的网络呢。
另外 v1.x 早就不维护啦，推荐升级到 v4 哈。
